examples/resources/grafana_cloud_observability_aws_account/resource.tf

```diff
@@ -0,0 +1,18 @@
+data "grafana_cloud_stack" "test" {
+  slug = grafana_cloud_stack.test.slug
+}
+
+resource "grafana_cloud_observability_aws_account" "test" {
+  stack_id = data.grafana_cloud_stack.test.id
+  name     = "my-aws-account"
+  role_arns = {
+    "my role 1a" = "arn:aws:iam::123456789012:role/my-role-1a",
+    "my role 1b" = "arn:aws:iam::123456789012:role/my-role-1b",
+    "my role 2"  = "arn:aws:iam::210987654321:role/my-role-2",
+  }
```

Comment:
I know we said we should support multiple arns at this level but it just feels weird looking at it ATM. Could we drop this down to a single one? If we end up needing to handle duplicates when migrating jobs to create the account we can create multiple account entities vs forcing this to work cleanly.

cc: @thepalbi because you brought it up originally

Reply:
yeah that's no problem! will do

Reply:
Yhea +1 on keeping 1 account entity <----> 1 ARN